cypress/integration/errors.spec.js

```diff
@@ -67,23 +67,17 @@ describe('test error handling', () => {
     cy.contains('Error 401: Was not able to query').should('be.visible');
   });
 
-  it('displays loading message', () => {
+  it.only('displays loading message', () => {
     cy.route({
       method: 'POST',
       url: 'bigquery/v2/projects/**/queries',
       status: 200,
       response: { jobComplete: false, jobReference: { jobId: 'jobId' } },
     }).as('getQueryResults');
-    cy.route({
```

Comment:
Just checking, why did this get removed? Was it not being used?

Reply:
For some reason, it wasn't ever firing. I suspect that something changed with how axios mocks stuff.  I didn't want to spend too much on this since this is behavior that will all go alway with Shelby's changes (and I manually tested and it works just fine)